keyserver/handler.go

```diff
@@ -28,18 +28,28 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 			p.Error(p.ErrorNameExisted)
 	}
 
-	tb, errCode := server.directory.Register(reg.Username, key)
+	ap, tb, errCode := server.directory.Register(reg.Username, key)
 	if errCode != p.Success {
 		server.Unlock()
 		return p.NewErrorResponse(errCode), p.Error(errCode)
 	}
 	server.tbs[reg.Username] = tb
 	server.Unlock()
 
+	// TODO: do we want to store the user policies into DB?
+
```

Comment:
By user policies you mean flags such as ``allowUnsignedKeyChange``? I think they should be stored in the DB as well since other clients need to use them.